SemanticUI/Modules/Dropdown/Select.elm

```diff
@@ -0,0 +1,230 @@
+module SemanticUI.Modules.Dropdown.Select exposing (Config, State, init, inline, select, selectMaybe, makeSelectToggle)
+
+{-| Refine a dropdown to act as a selection
+
+Example of `select` :
+
+    Select.select
+        (Select.init { identifier = "select1", onToggle = ToggleSelect1, onSelect = SetSelect1, choices =  [ Yes, No ]})
+        { dropdownState = model.select1, selectedValue = model.select1Selection }
+        div
+        []
+        (\makeSelect _ a -> makeSelect div [ class "text" ] [ text (toString a) ])
+
+Example of `selectMaybe` :
+
+    Select.selectMaybe
+        (Select.init { identifier = "select2", onToggle = ToggleSelect2, onSelect = SetSelect2, choices =  [ Yes, No ]})
+        { dropdownState = model.select2, selectedValue = model.select2Selection }
+        div
+        []
+        (\makeNoSelect -> makeNoSelect div [ class "text default" ] [ text "None Selected" ])
+        (\makeSelect _ a -> makeSelect div [ class "text" ] [ text (toString a) ])
+
+Example of inline select :
+
+    Select.select
+        (Select.init { identifier = "inline1", onToggle =  ToggleInline1, onSelect = SelectInline1, choices = [ "today", "this week", "this month" ]} |> Select.inline True)
+        model.inline1
+        model.inline1Selection
+        div
+        []
+        (\makeChoice isList c ->
+            makeChoice div
+                [ classList [ ( "active", isList && model.inline1Selection == c ) ] ]
+                [ text c ]
+        )
+
+-}
+
+import SemanticUI.Modules.Common exposing (Render, WrappedNode)
+import SemanticUI.Modules.Dropdown as Dropdown exposing (dropdown)
+import Html exposing (..)
+import Html.Attributes exposing (..)
+import Html.Events exposing (..)
+
+
+{-| Configuration for a select
+
+  - identifier - unique identifier
+  - onToggle - message to handle dropdown state changes
+  - toggleEvent - what casuse dropdown state change (default OnClick)
+  - readOnly - is the selection read only or not
+  - onSelect - messages to handle selection change (when clicking on choice)
+  - choices - the list of possible choices
+  - inline - should the selection be inline or not
+    (dropdown does not have "selection" class but "inline" and
+    "text" class is added to the unexpanded item)
+
+-}
+type alias Config r c s msg =
+    { r
+        | identifier : String
+        , onToggle : Dropdown.State -> msg
+        , toggleEvent : Dropdown.ToggleEvent
+        , readOnly : Bool
+        , onSelect : s -> msg
+        , choices : List c
```

Comment:
Would people be happy calling this `options`?

I believe the traditional html control would look something like

```
<select> 
  <option>A</option>
  <option>B</option>
</select>
```